ktor-legacy/ktor-client-legacy/build.gradle.kts

```diff
@@ -0,0 +1,11 @@
+description = "Ktor http client legacy code and deprecations"
```

Comment:
Is it OK if we will auto update all the copyright notices in the project after this is merged? Otherwise, files will be not only moved, but also changed which can make it harder to review.

Reply:
nvm, I updated all of them.